### DISCUSSION TITLE
Mapping information that is not closely aligned with the Darwin Core definitions of Symbiota fields

### DISCUSSION THREAD

**Discussion by msundue** [Symbiota/community-board#292] [General]
Our Ferns of Colombia project would like to include some collections information that does not clearly fit into any of the standard fields that we are familiar with. Can anyone suggest which fields would be most appropriate for the following?

1. The NSF award number and project title
2. The collection permit information under which the specimens were originally obtained

Thanks for any thoughts or suggestions!

**Comment by JCGiron** [comment]
I would suggest to reach out to the team at [SiB Colombia](https://biodiversidad.co/) which is the entity dealing with biodiversity data in the country. If you are managing your data using spreadsheets they for sure can tell you exactly what DwC fields to use for each of the pieces of information that you need to accomodate. How do those map onto Symbiota portals I do not know.
In particular, you can reach out to Camila Plata (cplata@humboldt.org.co), the lead content manager.
@camiplata @SIB-Colombia

**Comment by zygoballus** [comment]
@JCGiron - I had a similar question, but it's for a North and South America spider collection. What fields would be good to use for data such as:
- Condition of preserved specimen (for example 'carapace detached' or 'missing 3 legs')
- Type of storage (for example '6 dram vial' or '2 oz jar')
- Trap type (for example 'Lindgren funnel trap with lineatin lure')
- Trap deployment date

**Comment by camiplata** [comment]
Hi! Sorry for the delay answer.

@msundue here are my recomendations:

- The NSF award number:  this information is better suited for the metadata , section Project Data, fields Identifier
 and Funding
- The NSF project title: this information is better suited for the metadata , section Project Data, fields Title, you can also add this name using the Darwin Core field datasetName
- The collection permit information under which the specimens were originally obtained, until last time we check with the symbiota team most of the DwC extensions are not included in symbiota, so you will have to add this information after exporting the DwC file from symbiota. In Colombia the information about permits its documented on the DwC  extension GGBN permit: https://rs.gbif.org/extension/ggbn/permit_2022-08-08.xml

Which institution is going to publish this data? Your local node can help you making this changes.

**Comment by camiplata** [comment]
@zygoballus my recommendations for your information are:

- Condition of preserved specimen (for example 'carapace detached' or 'missing 3 legs'): you could use occurenceRemarks https://dwc.tdwg.org/terms/#dwc:occurrenceRemarks
- Type of storage (for example '6 dram vial' or '2 oz jar'): this is a information more related to the internal management of the collection so I always recommend to double check if it is really necessary to add it for the publication stage or not , but you could use the DwC term [preparations](https://dwc.tdwg.org/terms/#dwc:preparations) giving more information about the medium. Or add even more details using the DwC extension [GGBN preparations](https://rs.gbif.org/extension/ggbn/preparation.xml). Again symbiota does not includes most of the extensions so you may need to add the information after it is exported.
- Trap type (for example 'Lindgren funnel trap with lineatin lure'): you can use the following DwC fields [samplingProtocol](https://dwc.tdwg.org/list/#dwc_samplingProtocol) together with, [sampleSizeValue](https://dwc.tdwg.org/terms/#dwc:sampleSizeValue) and [sampleSizeUnit](https://dwc.tdwg.org/terms/#dwc:sampleSizeUnit), Symbiota may not have some of this fields included you will have to check directly with them.
- Trap deployment date: you can use eventDate field use as a date range for the days the trap was open aaaa-mm-dd/dd, or only one date if the trap has  open only for one day aaaa-mm-dd, you can add the specific time range using  [eventTime](https://dwc.tdwg.org/terms/#dwc:eventTime)

Hope this helps

**Comment by msundue** [comment]
Fantastic - thanks!

On Thu, Mar 23, 2023 at 1:26 PM CamiSilver ***@***.***> wrote:

> @zygoballus <https://github.com/zygoballus> my recommendations for your
> information are:
>
>    - Condition of preserved specimen (for example 'carapace detached' or
>    'missing 3 legs'): you could use occurenceRemarks
>    https://dwc.tdwg.org/terms/#dwc:occurrenceRemarks
>    - Type of storage (for example '6 dram vial' or '2 oz jar'): this is a
>    information more related to the internal management of the collection so I
>    always recommend to double check if it is really necessary to add it for
>    the publication stage or not , but you could use the DwC term
>    preparations <https://dwc.tdwg.org/terms/#dwc:preparations> giving
>    more information about the medium. Or add even more details using the DwC
>    extension GGBN preparations
>    <https://rs.gbif.org/extension/ggbn/preparation.xml>. Again symbiota
>    does not includes most of the extensions so you may need to add the
>    information after it is exported.
>    - Trap type (for example 'Lindgren funnel trap with lineatin lure'):
>    you can use the following DwC fields samplingProtocol
>    <https://dwc.tdwg.org/list/#dwc_samplingProtocol> together with,
>    sampleSizeValue <https://dwc.tdwg.org/terms/#dwc:sampleSizeValue> and
>    sampleSizeUnit <https://dwc.tdwg.org/terms/#dwc:sampleSizeUnit>,
>    Symbiota may not have some of this fields included you will have to check
>    directly with them.
>    - Trap deployment date: you can use eventDate field use as a date
>    range for the days the trap was open aaaa-mm-dd/dd, or only one date if the
>    trap has open only for one day aaaa-mm-dd, you can add the specific time
>    range using eventTime <https://dwc.tdwg.org/terms/#dwc:eventTime>
>
> Hope this helps
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BioKIC/symbiota-docs/discussions/292#discussioncomment-5409414>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABXVFQRTCWT7CJHBMPO34E3W5SBVFANCNFSM6AAAAAATV64GC4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Michael Sundue
The Pringle Herbarium
Dept. of Plant Biology - University of Vermont
Fernsoftheworld.com
he/him/his
